root/ADOverseas.py

```diff
@@ -1,114 +0,0 @@
-from flask import Flask, request, jsonify
-from datetime import datetime, timezone
-from apscheduler.schedulers.background import BackgroundScheduler
-from apscheduler.jobstores.sqlalchemy import SQLAlchemyJobStore
-from apscheduler.executors.pool import ThreadPoolExecutor, ProcessPoolExecutor
-from snippies import ad, app_log, config, db, scheduler_config
-import configparser
-import os
-
-app = Flask(__name__)
-scheduler = BackgroundScheduler(jobstores=scheduler_config.jobstores,
-								executors=scheduler_config.executors,
-								job_defaults=scheduler_config.job_defaults)
-scheduler.start()
-
-app_log.configure_logging()
-
-DATABASE = 'persistent/schedules.sqlite'
```

Comment:
the baseimage is designed around storing persistant data in `/config` no reason to break pre-existing [structure](https://github.com/linuxserver/docker-baseimage-alpine/blob/master/root/etc/s6-overlay/s6-rc.d/init-adduser/run#L50)